### PR TITLE
Update the endianness of the polynomial commitments to be big endian

### PR DESCRIPTION
**Pull request by g11tech**
Update the endianess of polynomal commitments to `big` endian in conjuction with 

- https://github.com/ethereum/EIPs/pull/7020

post ACD may 11 call discussion (Ref: https://discord.com/channels/595666850260713488/745077610685661265/1106250714415648849)

### CODE REVIEW

File: specs/deneb/polynomial-commitments.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Public functions MUST accept raw bytes as input and perform the required cryptog
 | `BYTES_PER_FIELD_ELEMENT` | `uint64(32)` | Bytes used to encode a BLS scalar field element |
 | `BYTES_PER_BLOB` | `uint64(BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB)` | The number of bytes in a blob |
 | `G1_POINT_AT_INFINITY` | `Bytes48(b'\xc0' + b'\x00' * 47)` | Serialized form of the point at infinity on the G1 group |
+| `KZG_ENDIANNESS` | `'big'` | The endianess of the field elements including blobs |
```
Comment (hwwhww) [inline specs/deneb/polynomial-commitments.md:80]:
```suggestion
| `KZG_ENDIANNESS` | `'big'` | The endianness of the field elements including blobs |
```

**Comment by etan-status** [comment]
In CL, represent these as `ByteVector` - they will retain endianness across layers this way.

CL shouldn't have to see into these commitments, it only has to pass the opaque data to the crypto lib

**Comment by hwwhww** [comment]
KZG test vectors: [general-kzg-4844-20230518.tar.gz](https://github.com/ethereum/consensus-specs/files/11508631/general-kzg-4844-20230518.tar.gz)

It would be nice if we could verify the KZG update correctness before the spec release. 🙏

/cc @jtraglia @asn-d6 @kevaundray

File: tests/generators/kzg_4844/main.py [hunk 2]
```diff
@@ -27,11 +27,11 @@ def expect_exception(func, *args):
```
Comment (asn-d6) [inline tests/generators/kzg_4844/main.py:27]: Hmm, do we also need to change the specification in `tests/formats/kzg`? If we are in a hurry, we can do it on a subsequent PR?

**Review by asn-d6 — APPROVED** [review]
LGTM! Thanks!

File: tests/generators/kzg_4844/main.py (hunk 2 above)
Comment (asn-d6) [reply tests/generators/kzg_4844/main.py:27]: Done in a7f865972

**Review by jtraglia — APPROVED** [review]
LGTM 👍

**Comment by kevaundray** [comment]
I'm encountering errors on the consensus-specs test in go-kzg-4844 -- Going to check in a minute to see if I made a wrong change

**Review by kevaundray — APPROVED** [review]

**Review by hwwhww — APPROVED** [review]

**Pull request merged**